app/views/ats/scorecards/_form.html.slim

```diff
@@ -90,41 +90,33 @@ ruby:
                               rows: 2,
                               placeholder: ""
 
-  .profile-card
-    .card
-      .card-header
-        span.card-title Summary
-      .card-body
-        span data-bs-toggle="tooltip" data-bs-placement="top" title="This is required question" Score
-        .row.g-1
-          .col-auto
-            = form.radio_button(:score, :irrelevant, required: true, class: "btn-check")
-            = form.label(:score_irrelevant, class: "btn btn-sm btn-outline-danger")
-              = render IconComponent.new(:mood_sad, icon_type: :filled)
-              | <br>Completely<br>irrelevant
-          .col-auto
-            = form.radio_button(:score, :relevant, required: true, class: "btn-check")
-            = form.label(:score_relevant, class: "btn btn-sm btn-outline-warning")
-              = render IconComponent.new(:mood_empty, icon_type: :filled)
-              | <br>Might be<br>irrelevant
-          .col-auto
-            = form.radio_button(:score, :good, required: true, class: "btn-check")
-            = form.label(:score_good, class: "btn btn-sm btn-outline-success")
-              = render IconComponent.new(:mood_smile, icon_type: :filled)
-              | <br>Good<br>candidate
-          .col-auto
-            = form.radio_button(:score, :perfect, required: true, class: "btn-check")
-            = form.label(:score_perfect, class: "btn btn-sm btn-outline-info")
-              = render IconComponent.new(:mood_happy, icon_type: :filled)
-              | <br>Perfect<br>candidate
-        .mt-3
-            | Open-ended feedback
-        .card
-          .card-body
-            = trix_editor form,
-                          :summary,
-                          placeholder: "",
-                          value: scorecard.summary
+  .card
+    .card-header
+      span.card-title Summary
+    .card-body
+      .d-flex
+        label.d-flex.form-label data-bs-toggle="tooltip" data-bs-placement="top" title="This is required question" Score
+      .form-selectgroup
+        - [%i[irrelevant mood_sad],
```

Comment:
may we compose one constant with score, icon and color and reuse it here and in scorecards helper?